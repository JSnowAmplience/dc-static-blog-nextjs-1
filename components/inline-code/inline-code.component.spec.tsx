import renderer from 'react-test-renderer';
import InlineCode from './inline-code.component';

describe('InlineCode', (): void => {
  it('should render some code as a code tag with styles', (): void => {
    const wrapper = renderer.create(<InlineCode value="import { something } from 'somewhere';" />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
