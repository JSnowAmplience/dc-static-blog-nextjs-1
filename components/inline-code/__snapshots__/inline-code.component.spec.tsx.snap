// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineCode should render some code as a code tag with styles 1`] = `
<code
  className="inline-code"
>
  import { something } from 'somewhere';
</code>
`;
